src/node_sockaddr-inl.h

```diff
@@ -12,7 +12,7 @@
 
 namespace node {
 
-namespace {
+namespace {  // NOLINT(build/namespaces)
```

Comment:
Can we remove the namespace here instead? The linter complains for a reason :)

Reply:
Absolutely, updated now.